ooiservices/app/uframe/assetController.py

```diff
@@ -204,6 +205,44 @@ def _associate_events(id):
 
     return result
 
+def _get_events_by_ref_des(ref_des):
+    #Create the container for the processed response
+    result = []
+    temp_dict = {}
+    #variables used in loop
+    platform = ""
+    mooring = ""
+    instrument = ""
+
+    #Get all the events to begin searching though...
+    uframe_obj = uFrameEventCollection()
+    data = uframe_obj.to_json()
+    for row in data:
+        try:
+            if  row['referenceDesignator']['subsite'] is not None:
+                platform = row['referenceDesignator']['subsite']
+            if row['referenceDesignator']['node'] is not None:
+                mooring = row['referenceDesignator']['node']
+            if row['referenceDesignator']['sensor'] is not None:  
+                instrument = row['referenceDesignator']['sensor']
+            concat_ref_des =  '-'.join([platform, mooring, instrument])
+            if concat_ref_des ==  ref_des:
+                temp_dict['id'] = row['eventId']
+                temp_dict['ref_des'] = concat_ref_des
+                temp_dict['start_date'] = num2date(float(row['startDate'])/1000, units='seconds since 1970-01-01 00:00:00', calendar='gregorian') 
```

Comment:
@Bobfrat here's how the date comes out of the assetController now, basically what you had